applications/nrf5340_audio/src/audio/audio_system.c

```diff
@@ -113,23 +115,17 @@ static void audio_headset_configure(void)
 static void encoder_thread(void *arg1, void *arg2, void *arg3)
 {
 	int ret;
-	uint32_t blocks_alloced_num;
-	uint32_t blocks_locked_num;
-
-	int debug_trans_count = 0;
-
+	uint32_t q_num_used;
 	char pcm_raw_data[FRAME_SIZE_BYTES];
-
-	static size_t pcm_block_size;
 	static uint32_t test_tone_finite_pos;
+	int debug_trans_count = 0;
 
 	while (1) {
 		/* Don't start encoding until the stream needing it has started */
 		ret = k_poll(&encoder_evt, 1, K_FOREVER);
-		struct audio_data audio_frame = {0};
-		struct net_buf *frame_buf = net_buf_alloc(&fifo_rx_pool, K_NO_WAIT);
+		struct net_buf *audio_frame = net_buf_alloc(&q_rx_pool, K_NO_WAIT);
```

Comment:
Discuss: Some places `audio_q` is used and in other places just `q`. I prefer the first, but it should be consistent.

Reply:
Changed all to audio_q